cpp/ps0003/Task3.cpp

```diff
@@ -4,18 +4,20 @@ using namespace std;
 
 int main()
 {
+    // prompting user for number 2/2
     float x;
 
     cout << "Enter your number: \n";
     cin >> x;

```

Comment:
- [x] declared a variable to store users input
- [x] used `std::cin` to prompt user for the number